### DISCUSSION TITLE
SSE capability

### DISCUSSION THREAD

**Discussion by andrewfg** [mockoon/mockoon#979] [General]
Does Mockoon have an SSE capability? i.e. is it possible for the test client to open a Server Side Event session with the Mockoon server, and have Mockoon push JSON payloads to the client at (say) irregular intervals?

**Comment by 255kb** [comment]
This is not supported by Mockoon which is focusing on answering and not doing any calls (except for the proxy part). 
WebSocket support will probably come at one point but SSE I am not sure. We add features mostly based on popularity and user requests.

**Comment by andrewfg** [comment]
I think that if you do Websocket, you can also quite easily do SSE as well. The only difference is the transport layer. Also your point about answering and not doing calls is not really the issue, since in SSE and Websockets, the server is essentially delivering a series of ‘answer’ payloads to the client, at random time delay intervals after the initial connect call was made.

**Comment by 255kb** [comment]
Ok I get it. I just had to deal with OpenAI SSE, so I see why you would need this. 
While not very different from websocket I think it would be a slightly different implementation, with probably a different UI too. 

Could you open an issue for this so it can be tracked properly, and maybe upvoted, etc. 
Thank you!

**Reply by andrewfg** [reply]
^
Done .. https://github.com/mockoon/mockoon/issues/990
